dwave/cloud/testing/mocks.py

```diff
@@ -222,6 +222,70 @@ def qpu_pegasus_solver_data(m: int,
 
     return structured_solver_data(**params)
 
+def qpu_access_time_data(qpu: str = 'Advantage_system4.1', **kwargs) -> dict:
+    """Mock QPU solver data with qpu_access_time.
+
+    Args:
+        qpu:
+            Name of the original solver that provided timing data used here.
+        **kwargs:
+            Solver properties passed down to :meth:`.structured_solver_data`.
+
+    """
+
+    timing_data_advantage41_2022_8 = {'version': '1.0.0',
+        'typical_programming_time': 14072.88,
+        'reverse_annealing_with_reinit_prog_time_delta': 0.0,
+        'reverse_annealing_without_reinit_prog_time_delta': 5.55,
+        'default_programming_thermalization': 1000.0,
+        'default_annealing_time': 20.0,
+        'readout_time_model': 'pwl_log_log',
+        'readout_time_model_parameters': [0.0, 0.7699665876947938, 1.7242758696010096,
+                2.711975459489206, 3.1639057672764026, 3.750276915153992, 1.539131714800995,
+                1.8726623164229292, 2.125631787097315, 2.332672340068556, 2.371606651233025,
+                2.3716219271760215],
+        'qpu_delay_time_per_sample': 20.54,
+        'reverse_annealing_with_reinit_delay_time_delta': -4.5,
+        'reverse_annealing_without_reinit_delay_time_delta': -1.5,
+        'default_readout_thermalization': 0.0,
+        'decorrelation_max_nominal_anneal_time': 2000.0,
+        'decorrelation_time_range': [500.0, 10000.0]}
+
+    timing_data_2000q6_2022_8 = {'version': '1.0.0',
+        'typical_programming_time': 10536.81,
+        'reverse_annealing_with_reinit_prog_time_delta': 0.0,
+        'reverse_annealing_without_reinit_prog_time_delta': 359.58,
+        'default_programming_thermalization': 1000.0,
+        'default_annealing_time': 20.0,
+        'readout_time_model': 'pwl_log_log',
+        'readout_time_model_parameters': [0.0, 3.30984300471607, 2.2975416678181597,
+            2.2975416678181597],
+        'qpu_delay_time_per_sample': 20.54,
+        'reverse_annealing_with_reinit_delay_time_delta': 586.38,
+        'reverse_annealing_without_reinit_delay_time_delta': -5.0,
+        'default_readout_thermalization': 0.0,
+        'decorrelation_max_nominal_anneal_time': 2000.0,
+        'decorrelation_time_range': [500.0, 10000.0]}
```

Comment:
Let's extract these two in a helper function `qpu_problem_timing_data` (get rid of `qpu_access_time_data`), and then simply use that function from the unittests below as:
```
C16 = StructuredSolver(data=mocks.qpu_chimera_solver_data(16, 
    problem_timing_data=qpu_problem_timing_data(generation='2000q')), client=None)
```
Or split `qpu_problem_timing_data` in `qpu_chimera_problem_timing_data` and `qpu_pegasus_problem_timing_data`?

Reply:
Implemented first option